data/compliance_profiles/schema.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "description": "",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "compliance_markup::compliance_map::percent_sign": {
+      "type": "string",
+      "minLength": 1
+    },
+    "compliance_markup::compliance_map": {
+      "type": "object",
+      "properties": {
+        "version": {
+          "type": "string",
+          "minLength": 5
+        }
+      },
+      "patternProperties": {
+        "^((?!version).)*$": {
+          "type": "object",
+          "additionalProperties": false,
+          "patternProperties": {
+            "^\\w+(?:::\\w+)+$": {
+              "type": "object",
+              "properties": {
```

Comment:
@heliocentric Can you make sure that the properties listed here contain all the possibilities that the mapper can handle?